### ISSUE TITLE
Can't find tool_objectfs settings

### ISSUE THREAD

**Issue by jsecchiero** [catalyst/moodle-tool_objectfs#110]
Hi there,
i have moodle version 3.4.1 and i'm trying to make this plugin work with s3. I have followed the readme and installation seems ok but still missing the settings menu. Going to admin/tools.php i see:

- Object storage file system | 2017111700

and in /admin/plugins.php:

- Amazon's SDK for PHP local_aws

i have tried both master and release 2017111700 for tool_objectfs.
i have tried both master and release 2017071501 for aws plugin.

I'm missing something?
Thanks
J

**Comment by mattporritt** [comment]
I have not seen this issue before.
What happens when you attempt to go to: admin/tool/objectfs/index.php ?

**Comment by jsecchiero** [comment]
Section error
![image](https://user-images.githubusercontent.com/17151394/46332495-febd4180-c61c-11e8-9cb7-2c32b9386c60.png)

I have tried with moodle version 3.3 and now i see the options. This plugin is incompatible with 3.4?

**Comment by alferx** [comment]
Hi, as a suggestion,
is the plugin compliant with PHP 7 ?
(Moodle 3.4 or later requires at least PHP 7)

**Comment by mattporritt** [comment]
Yes the plugin works with PHP7 and above. We user it with many clients on PHP7
